data.json

```diff
@@ -511,8 +511,21 @@
         "name": "WebStorm",
         "icon_path": "icon_WebStorm.png",
         "emoji_name": "webstorm",
-        "channel_name": "webstorm",
-        "category_name": "Products",
+        "channels": [
+            {
+                "name": "webstorm-announcements",
+                "description": "Product announcements and updates"
+            },
+            {
+                "name": "webstorm-general",
+                "description": "General discussions about WebStorm"
+            },
+            {
+                "name": "webstorm-help-hub",
+                "description": "Get help with WebStorm"
```

Comment:
I think we might need a way to mark that this channel is a forum rather than a text channel? This likely has quite a few implications throughout the bot where we're looking at text channels only.

Reply:
that's a good point, I completely forgot about that part